test/podmontest/insv.sh

```diff
@@ -1,22 +1,25 @@
 #!/bin
-#
-# Copyright (c) 2021. Dell Inc., or its subsidiaries. All Rights Reserved.
+# Copyright (c) 2021-2022 Dell Inc., or its subsidiaries. All Rights Reserved.
 #
 # Licensed under the Apache License, Version 2.0 (the "License");
 # you may not use this file except in compliance with the License.
 # You may obtain a copy of the License at
 #
 # http://www.apache.org/licenses/LICENSE-2.0
 #
-#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
 
 SCRIPTDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
 instances=${instances:-4}
 ndevices=${ndevices:-0}
 nvolumes=${nvolumes:-4}
 zone=${zone:-""}
 storageClassName=${storageClassName:-vxflexos-retain}
-image="$REGISTRY_HOST:$REGISTRY_PORT/podmontest:v0.0.54"
+image="10.247.98.98:5000/podmontest:v0.0.54"
```

Comment:
Remove the IP, we must care full, internal IPs should never be on a PR

Reply:
fixed